packages/provider/src/json-rpc-provider.ts

```diff
@@ -342,6 +342,38 @@ export class JsonRpcProvider implements AbstractProvider {
     } as PaginableBlockTransactions
   }
 
+  /**
+   * Converts an object to Node
+   * @param {any} maybeNode - a Node object returned from API
+   * @returns {Node} - Node converted from maybeNode
+   * */
+  static asNode(maybeNode: any): Node {
+    const {
+      address,
+      chains,
+      jailed,
+      output_address,
+      public_key,
+      reward_delegators,
+      service_url,
+      status,
+      tokens,
+      unstaking_time,
+    } = maybeNode
+    return {
+      address,
+      chains,
+      jailed,
+      outputAddress: output_address,
+      publicKey: public_key,
+      rewardDelegators: reward_delegators,
+      serviceUrl: service_url,
+      status,
+      stakedTokens: tokens.toString(),
```

Comment:
the old implementation used:

staked_tokens ?? '0', are we changing the behavior here?

Reply:
I respected the old implementation in `getNode`.  I'm sure nobody has been using `getNodes` for the following reasons:
- it sent a query to `/v1/query/apps`, meaning `getNodes` never returned nodes 
- Node doesn't have `staked_tokens`.  It's in App.  Node has `tokens`.
- Every node returned from `/v1/query/apps` is expected to have a number in `tokens`.  We will not see a nullish value in `tokens`.